nixos/doc/manual/configuration/subversion.xml

```diff
@@ -93,23 +94,23 @@ services.httpd.extraModules = [
     </para>
     <para>
 <screen>
-<prompt>$ </prompt> svn create REPO_NAME
```

Comment:
The spaces are supposed to be in prompt so that it is not copied.

Reply:
i think it won't let copy even now: 
![image](https://user-images.githubusercontent.com/75213/101277614-58ea5c00-37be-11eb-9b91-7e29ec3a8c26.png)

(correct me if i didn't understood you correctly)

but i would feel much help if i can copy any code formatted block for re-use.

Reply:
In the picture, you can see that the space before the command is selected. But the space is not a part of the command so it should not be copied.

To achieve that, the space needs to be inside the prompt:

```docbook
<prompt>$ </prompt>fooo
```

instead of outside it like you have here:

```docbook
<prompt>$</prompt> fooo
```

Reply:
got it, corrected.